homeassistant/components/tuya/strings.json

```diff
@@ -523,6 +523,9 @@
       "phase_c_voltage": {
         "name": "Phase C voltage"
       },
+      "add_ele": {
+        "name": "Current energy level"
```

Comment:
Just so i understand, this means the circuit breaker has built in battery backup? And this is the charge level of that Battery?

Reply:
It indicates the current phase of the circuit breaker's energy consumption, reporting the energy consumption for the next phase after completion. So current energy level is better to explain it.

Reply:
Current as in Now/Active or Current as in Ampere?
Phase as in "current step of multi step process" or as in one of the cables of a 3 phase system?
Level as in the tank is filled to half level or ?

Ie, its still very unclear what this means.

Reply:
How about "Current phase energy"? Here is the explanation:

The phrase "Current phase energy" is a suitable translation because it combines the key elements of the original Chinese expression: 当前阶段通断器设备的计量电量值

"Current" indicates the present state or phase.
"Phase" refers to a specific step or stage within a larger process.
"Energy" represents the measured electrical quantity or power consumption.

Let me know if it works? :) @elupus

Reply:
Or "Current electricity consumption“ ?

Reply:
We have used "Current consumption" on the past for instantaneous Watt or Amperage..
We should avoid "Phase" if it does not mean one of the cables in a 3 phase  #system.

Reply:
OK，"Current consumption" sounds good, just update and commit it. Please help to check if there is any other issues. :)